src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/sourceField.tsx

```diff
@@ -36,6 +36,7 @@ class SourceField extends React.Component<Props, State> {
     activeSuggestion: 0,
     showSuggestions: false,
     hideCaret: false,
+    help: t('Where to look. In the simplest case this can be an attribute name.'),
```

Comment:
Now that the help prop is always defined, I think it's a good idea to remove the optional parameter of the State 😉